network/wsNetwork.go

```diff
@@ -826,6 +826,28 @@ func (wn *WebsocketNetwork) checkIncomingConnectionLimits(response http.Response
 	return http.StatusOK
 }
 
+// checkProtocolVersionMatch test ProtocolAcceptVersionHeader and ProtocolVersionHeader headers from the request/response and see if it can find a match.
+func (wn *WebsocketNetwork) checkProtocolVersionMatch(otherHeaders http.Header) (matchingVersion string, otherVersion string) {
```

Comment:
Why does this function return `otherVersion`?  It doesn't seem to be meaningfully used by its caller.

Reply:
The usage of `otherVersion` is for the logging/response text. This is the only indication we have that the mismatch was with the `ProtocolVersionHeader`. As an alternative, we can try to re-extract the value from the headers, but I'm not sure if that would improve the code readability.